lib/vagrant-packer-plugin/command.rb

```diff
@@ -106,9 +113,10 @@ def execute
         end
 
         puts "[packer-info] Packaging berkshelf recipes..."
-        # TODO - make it parametric
-        Berkshelf::Cli.start(["package",@params.cookbooks_url.split('/')[-1]])
 
+        Berkshelf::Cli.start(["package",@params.cookbooks_url.split('/')[-1]])
+        # TODO - consider also params.berksfile, but not working yet
```

Comment:
you added -B to the OptParser. Not  -b! https://github.com/Alfresco/alfresco-spk/blob/d284d2c49284d37d185a3bd2225301ab9d8119a4/lib/vagrant-packer-plugin/command.rb#L33